kratos/input_output/logger_output.cpp

```diff
@@ -53,7 +56,12 @@ namespace Kratos
             switch (message_severity)
             {
             case LoggerMessage::Severity::WARNING:
-                if (mOptions.Is(WARNING_PREFIX)) mrStream << "[WARNING] ";
+                if (mOptions.Is(WARNING_PREFIX)) {
+                    #if defined(KRATOS_COLORED_LOGGING)
+                    mrStream << KYEL;
```

Comment:
where is the KYEL defined?

Reply:
in "utilities/color_utilities.h" - see include

Reply:
was already there.